content/gotraining-2019-05-melbourne.slide

```diff
@@ -0,0 +1,1050 @@
+Go training
+Melbourne, May-June 2019
+
+Angelo Perera
+Julia Ogris
+Ryan O'Kane
+Sai Kiran Gummaraj
+
+* Introduction
+
+
+* Agenda
+
+- Introduction to Go
+- Go language fundamentals
+- Go standard library
+- Go tool chain
+- Third party Go libraries commonly used
+- Project work
+
+Find Slides at [[https://gotraining.appspot.com]] ([[https://github.com/anz-bank/go-slides][source]] on GitHub)
+
+
+* Day 1 - Schedule
+
+*Morning:* Introduction to Go
+
+- Overview and history
+- Demo
+
+*Afternoon:* Hello world and first PR
+
+- Interlude: towards continuous deployment (CD)
+- Go language fundamentals
+- First lab
+
+
+* History and Overview
+
+
+* History of Go
+
+.image _img/xkcd_compiling.png
+.caption [[https://xkcd.com/303/]]
+
+2007: Robert Griesemer, Rob Pike, and Ken Thompson start working on Go at Google
+2008: Russ Cox (go mod) and Ian Taylor (gcc frontend) join
+2009: Go goes Open Source
+2012: Go version 1.0
+
+
+* What is Go?
+
+- A simple language that is easy to learn and read
+- Statically typed, but with a dynamic feel
+- Compiled to native machine code, but has a fast development cycle
+- Language-level concurrency features
+- Comprehensive and clear standard library
+- Great tools
+- Open source
+
+(Source: [[https://talks.golang.org/2012/zen.slide#2][Go and the Zen of Python]])
+
+
+* What does Go look like?
+
+Hello world in Go
+
+	package main
+
+	import (
+		"fmt"
+	)
+
+	func main() {
+		fmt.Println("Hello, ಬೆಂಗಳೂರು")
```

Comment:
Non english text. We need to change this to something like `fmt.Println("Hello, Melbourne")`

Reply:
Done (plus some emojis to prove unicode-ability)